Atomic/backends/_ostree.py

```diff
@@ -26,18 +26,20 @@ def backend(self):
 
     def _make_container(self, info):
         container_id = info['Id']
-
         runtime = self.syscontainers.get_container_runtime_info(container_id)
-
         container = Container(container_id, backend=self)
         container.name = container_id
+        container.command = info['Command']
         container.id = container_id
+        container.image_name = info['Image']
+        container.image_id = info['ImageID']
         container.created = info['Created']
-        container.status = runtime['status']
+        container.status = container.state = runtime['status']
```

Comment:
Depending on how the struct information is gather (i.e. inspect vs containers), they have State and/or Status and they differ.  In ostree, we set both to be the same and handle it.